<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMainWidgt</class>
 <widget class="QWidget" name="FMainWidgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FMainWidgt</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="proFileTableVw"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>请选择要添加pro文件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="appendBtn">
           <property name="text">
            <string>Add...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>qmake路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="qmakePathLndt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setQmakePathBtn">
           <property name="text">
            <string>设置...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Qt  版 本：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="qtVersionLndt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>make 路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="makePathLndt">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setMakePathBtn">
           <property name="text">
            <string>设置...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>编 译路径：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="compiledPathLdt"/>
         </item>
         <item>
          <widget class="QPushButton" name="setCompilePathBtn">
           <property name="text">
            <string>Open...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>编译版本：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="compilTypeCmBx">
           <item>
            <property name="text">
             <string>Release</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Debug</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopCompiledBtn">
           <property name="text">
            <string>停止编译</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="compilingBtn">
           <property name="text">
            <string>编译</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="logTabWgt">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="normalLogTab">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="normalTextBw"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="errorLogTab">
       <attribute name="title">
        <string>Error</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="errorTextBw"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
